### PR TITLE
Twitter sentiments api

### PR DESCRIPTION
**Pull request by mina-kleid**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "react-dom": "^0.14.7",
     "react-router": "^2.0.0",
     "react-router-bootstrap": "^0.20.1",
+    "request": "^2.69.0",
```
Comment (yashha) [inline package.json:69]: Did you use it?

**Comment by yashha** [comment]
Looks good, but how do we handle getting sentiments about certain characters?
They would overwrite themselves.

File: package.json (hunk 1 above)
Comment (mina-kleid) [inline package.json:69]: no but it was missing from wikirequest.js line 3 and threw an error

**Comment by mina-kleid** [comment]
@yashha not really, since each instance on the client has its own store (its per tab in the browser, not per session). so if the user navigates to other character it will be reloaded and overwritten, which is correct.
If the user opens it in a new tab, then its a new instance with its own new store.
Same as happening with characters and characters store

**Pull request merged**